split/resource_split_split_test.go

```diff
@@ -28,6 +28,8 @@ func TestAccSplitSplit_Basic(t *testing.T) {
 						"split_split.foobar", "description", "my split description"),
 					resource.TestCheckResourceAttrSet(
 						"split_split.foobar", "traffic_type_id"),
+					resource.TestCheckResourceAttrSet(
```

Comment:
It would be great to add a testcase in this file that adds tags to the feature flag instead of just checking if it is read.